tests/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtensionTest.php

```diff
@@ -264,8 +264,8 @@ public function testCompositeIdentifiers()
             ->setParameter('foo', 1);
 
         $queryNameGenerator = $this->prophesize(QueryNameGeneratorInterface::class);
-        $queryNameGenerator->generateJoinAlias('item')->shouldBeCalled()->willReturn('item_2');
-        $queryNameGenerator->generateJoinAlias('label')->shouldBeCalled()->willReturn('label_2');
+        $queryNameGenerator->generateJoinAlias('compositeItem')->shouldBeCalled()->willReturn('item_2');
+        $queryNameGenerator->generateJoinAlias('compositeLabel')->shouldBeCalled()->willReturn('label_2');
```

Comment:
This change is wrong, `item` and `label` are correct I think (at least according to failing tests).

Reply:
Nop, cause this call is made through [QueryBuilderHelper::addJoinOnce](https://github.com/api-platform/core/pull/1324/files#diff-44f9dbe5365de635dbe64677eb371d9cR48) from association name.

I don't understand why travis is failing…on a non-existing unit test 😮 (failing method `testCompositeIdentifiersWithoutAssociation` does not exist in `FilterEagerLoadingExtensionTest`)

Reply:
I'll try to squash my changes to force travis to rebuild its environment

Reply:
I added this recently maybe a bad rebase? 
https://github.com/api-platform/core/blob/d18ec0da1ac52f92ca755ca275cd0853d0972878/tests/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtensionTest.php

Reply:
Rebase missing, I fixed it, thanks @soyuka